src/amo/components/GetFirefoxButton/index.js

```diff
@@ -37,12 +38,14 @@ export type Props = {|
 
 type InternalProps = {|
   ...Props,
+  _base64url?: typeof base64url,
   i18n: I18nType,
   userAgentInfo: UserAgentInfoType,
 |};
 
 export const GetFirefoxButtonBase = (props: InternalProps) => {
   const { addon, buttonType, className, i18n, userAgentInfo } = props;
+  const _base64url = props._base64url || base64url;
```

Comment:
I guess `props._base64url` is only added here for testing purposes?

Reply:
Correct. I wanted to be able to add a test to verify that we're calling `base64url.encode`.